sdcm/nemesis.py

```diff
@@ -2347,10 +2347,18 @@ def is_streaming_started():
         time.sleep(random.randint(10, 600))
 
         self.log.debug('Interrupt the task by hard reboot')
-        self.target_node.reboot(hard=True, verify_ssh=True)
-        streaming_thread.join(60)
+        try:
+            self.target_node.reboot(hard=True, verify_ssh=True)
+            streaming_thread.join(60)
+        except Exception as err:  # pylint: disable=broad-except
+            if task == 'decommission':
+                self.log.debug(f'Failed to reboot the target node, decommission might complete')
+            else:
+                raise
```

Comment:
raise what? maybe `raise err`? or raise a better message for user to be able to catch why it failed?

Reply:
open `raise` is o.k. it would raise the current exception that was caught